src/common/primitive_attr_quant.hpp

```diff
@@ -244,6 +244,7 @@ struct zero_points_t : public c_compatible {
 
     // arg-specific checks
     bool common(int arg) const { return get_mask(arg) == 0; }
+    bool per_oc(int arg) const { return get_mask(arg) == 2; }
```

Comment:
Suggest not to introduce this as primitives may interpret `mask == 2` not necessarily as `per_oc` and rely on `get_mask()` directly.

Reply:
@dzarukin `get_mask()` is private, you suggest to change it to public?

Reply:
I see, wouldn't think it's in `private`... Let's use `per_dim_1` then, I'll replace it as a part of future refactor later. Thanks for checking.